front/app/containers/Admin/projects/project/nativeSurvey/FormResults/TextResponses.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, colors, Text } from '@citizenlab/cl2-component-library';
+import { useVirtualizer } from '@tanstack/react-virtual';
+import Divider from 'components/admin/Divider';
+import React from 'react';
+import styled from 'styled-components';
+import { useIntl } from 'utils/cl-intl';
+import messages from './messages';
+
+const Item = styled.div<{ start: number }>`
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  transform: translateY(${(props) => props.start}px);
+`;
+
+type TextResponsesProps = {
+  textResponses: {
+    answer: string;
+  }[];
+};
+
+const TextResponses = ({ textResponses }: TextResponsesProps) => {
+  const parentRef = React.useRef(null);
+  const { formatMessage } = useIntl();
+
+  // The virtualizer
+  const { measureElement, getTotalSize, getVirtualItems } = useVirtualizer({
```

Comment:
Using a virtualized list means we can display a large number of items without performance issues.